{{ cookiecutter.repo_name }}/.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -13,10 +13,16 @@ assignees: ""
 - [ ] License information is verified as correct. If you are unsure, please comment below.
 - [ ] Locally rendered documentation contains all appropriate pages, including API references (check no modules are
   missing), tutorials, and other human written text is up-to-date with any changes in the code.
-- [ ] Installation instructions in the README, documentation and on the website (e.g., diffpy.org) are updated and
-  tested
-- [ ] Successfully run any tutorial examples or do functional testing in some other way.
+- [ ] Installation instructions in the README, documentation and on the website (e.g., diffpy.org) udpated.
+- [ ] Successfully run any tutorial examples or do functional testing with the latest Python version
 - [ ] Grammar and writing quality have been checked (no typos).
 
 Please mention @sbillinge when you are ready for release. Include any additional comments necessary, such as
-version information and details about the pre-release.
+version information and details about the pre-release here:
+
+### Post-release checklist
+
+Before closing this issue, please complete the following:
+
+- [ ]  Run tutorial examples and conduct functional testing using the installation guide in the README.
+- [ ]  Documentation (README, tutorials, API references, and websites) is deployed without broken links or missing figures.
```

Comment:
before closing, ensure to double check all user-facing content - suggested by @cadenmyers13